cmd/util.go

```diff
@@ -40,10 +41,12 @@ func newUtilCommand() *cobra.Command {
 	flags.LoggerFlags(generateJWTCmd.Flags())
 	flags.AuthOIDCClaimUsername(generateJWTCmd.Flags())
 	flags.AuthOIDCClaimGroups(generateJWTCmd.Flags())
+	flags.AuthOIDCClaimRoles(generateJWTCmd.Flags())
 	flags.AuthOIDCCertKeyFile(generateJWTCmd.Flags())
 
 	generateJWTCmd.Flags().StringVar(&sub, "sub", "", "username and tenant added in claims")
-	generateJWTCmd.Flags().StringArrayVar(&scope, "scope", []string{}, "roles and permissions added in claims")
+	generateJWTCmd.Flags().StringArrayVar(&scope, "scope", []string{}, "permissions added in claims")
+	generateJWTCmd.Flags().StringArrayVar(&roles, "qkm-user-roles", []string{}, "roles added in claims")
```

Comment:
Value `"qkm-user-roles"` should be taken from flags instead of hard-coding it here